app/backend/src/route/index.ts

```diff
@@ -0,0 +1,6 @@
+import { healthCheck } from './healthCheck';
+import { userRoute } from './user';
+
+const routes = [userRoute, healthCheck];
```

Comment:
why not just export default?

`module.exports = {
    userRoute,
    healthCheck
}`

Or

`export const routes = {
    userRoute,
    healthCheck
}`